src/cdk/datepicker/calendar-view.ts

```diff
@@ -28,5 +28,5 @@ export abstract class CalendarView<D> {
   abstract selected: D | null;
 
   /** Emits when a new date is selected. */
-  @Output() abstract readonly selectedChange = new Subject<D | null>();
+  abstract readonly selectedChange = new Subject<D | null>();
```

Comment:
All of the extended classes of `CalendarView` have the `selectedChange` field implemented as an `EventEmitter`, which is an extension of `Subject`. The `Datepicker` CDK will also have its own `selectedChange`. Do we need another initialization of a `selectedChange` here?

Reply:
I think it's fine to do `= EventEmitter<D | null>()` here, that will allow us to also do the cleanup in `ngOnDestroy` here instead of making each subclass do it